omnipaxos/src/sequence_paxos/follower.rs

```diff
@@ -160,23 +160,25 @@ where
     fn reply_accepted(&mut self, n: Ballot, accepted_idx: usize) {
         match &self.latest_accepted_meta {
             Some((round, outgoing_idx)) if round == &n => {
-                let PaxosMessage { msg, .. } = self.outgoing.get_mut(*outgoing_idx).unwrap();
-                match msg {
-                    PaxosMsg::Accepted(a) => {
-                        a.accepted_idx = accepted_idx;
-                    }
-                    _ => panic!("Cached idx is not an Accepted Message<T>!"),
+                if let Message::SequencePaxos(PaxosMessage {
+                    msg: PaxosMsg::Accepted(a),
+                    ..
+                }) = self.outgoing.get_mut(*outgoing_idx).unwrap()
+                {
+                    a.accepted_idx = accepted_idx
+                } else {
+                    panic!("Cached idx is not an Accepted Message<T>!");
```

Comment:
we shouldn't panic if something is wrong with the cache. We should just append the message and update the cache.